templates/resources/cluster_aks.md.tmpl

```diff
@@ -12,7 +12,7 @@ description: |-
 ## Example Usage
 
 ```terraform
-data "spectrocloud_cloudaccount_aws" "account" {
+data "spectrocloud_cloudaccount_azure" "account" {
```

Comment:
@paraglade this is the template that needs to be changed, however I changed it, so all good!